rocks/consul/rockcraft.yaml

```diff
@@ -0,0 +1,51 @@
+name: consul
+summary: Hashicorp Consul
+license: Apache-2.0
+description: |
+  Ubuntu distribution of Hashicorp Consul
+version: 'v1.19.2'
```

Comment:
I am just wondering whether the tag should have `v` as prefix (given version command prints with `v` as prefix). Looked at rabbitmq, traefik, vault rocks and they have version without `v`. 
In upstream docker images its a mix like consul tags does not have `v` and traefik tags have `v`.

I am not sure if there is preferred way for the rocks under github.com/canonical. Given rabbitmq, traefik, vault rocks does not have `v` as prefix, lets go with similar notation just `1.19.2`

What do you think?

Reply:
I hear you. The version on the Consul repo is literally tagged `v1.19.2`. I'm fine to drop the `v` and do `1.19.2` if it doesn't make any difference to diverge from the exact release tag on the Consul repo.